content/en/logs/guide/build-custom-reports-using-log-analytics-api.md

```diff
@@ -0,0 +1,667 @@
+---
+title: Build custom reports using Log Analytics API
+kind: guide
+further_reading:
+- link: "/logs/explorer/analytics/?tab=timeseries#overview"
+  tag: "Documentation"
+  text: "Learn more about Log Analytics"
+- link: "/api/v2/logs/#aggregate-events"
+  tag: "Documentation"
+  text: "Syntax for Log Analytics API"
+- link: "/logs/guide/collect-multiple-logs-with-pagination/?tab=v1api"
+  tag: "Documentation"
+  text: "Collect multiple logs with Pagination" 
+---
+
+
+## Overview
+
+You may want to build custom reports and dashboards for your team by combining information from your business and other services alongside log data. To quickly build such reports with metrics from your log data, use the [Log Analytics API][1].
+
+The following examples are covered in this guide:
+
+* [Getting counts](#getting-counts)
+* [Getting stats](#getting-stats)
+* [Getting percentiles](#getting-percentiles)
+* [Multiple groupbys, unique counts, and metrics](#multiple-groupbys-unique-counts-and-metrics) 
+
+## Prerequisites
+
+Since this guide describes usage of the API, you will need an API key and an application key from an admin user. These are available in your [Datadog account API key page][2].
+
+Throughout this article, you will need to replace all occurrences of `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your Datadog API key and your Datadog application key, respectively.
+
+This guide also assumes that you have a terminal with `CURL`. 
+
+## Examples
+
+### Getting counts
+
+{{< tabs >}}
+{{% tab "Table" %}}
+
+With the following API call, you can build a `table` with `count` of log events grouped by the field `status`. The `type` must be `total`.
+
+**API call:**
```

Comment:
Should we specify a top value here just so that users have an example on how to define it?

We should likely add it to all examples and then have a specific table example with different top values? (that would be advanced though).